### ISSUE TITLE
Lint duration in v3

### ISSUE THREAD

**Issue by thomas88** [amilajack/eslint-plugin-compat#193]
When using v3 the lint duration increases significantly from 2min20sec to 5min:

```sh
# v3
$ yarn lint:js  279.63s user 9.65s system 96% cpu 5:00.14 total

# v2
$ yarn lint:js  145.92s user 7.04s system 109% cpu 2:20.31 total
```

**Comment by amilajack** [comment]
This is likely because we significantly increased the number of API's we check for compatibility. I'll look into ways to optimize this. Thanks for reporting this.

**Comment by TrevorBurnham** [comment]
👋 Howdy! I've opened up two PRs that should cut linting time significantly: #245, #246

@thomas88 If you get a chance, please try testing those branches against your project and let me know how much of a difference they make.

**Comment by saviomuc** [comment]
Hi @TrevorBurnham and @thomas88. 

Works for me. I tested the latest version and got these metrics 👏 :

2.7.0 >  `yarn lint:js  108.07s user 4.37s system 113% cpu 1:38.74 total`
3.3.0-1 > `yarn lint:js  111.01s user 4.75s system 113% cpu 1:41.76 total`
